ondemand/install.sh

```diff
@@ -7,6 +7,185 @@ log_info() {
   printf "\n\e[0;35m $1\e[0m\n\n"
 }
 
+build_passenger() {
+
+  PASS_VERSION='6.0.7'
+  NGINX_VERSION='1.18.0'
+
+  wget -O $BUILD_DIR/passenger.tar.gz https://github.com/phusion/passenger/releases/download/release-$PASS_VERSION/passenger-$PASS_VERSION.tar.gz
+  cd $BUILD_DIR
+  tar xf passenger.tar.gz
+  wget -O $BUILD_DIR/nginx.tar.gz http://nginx.org/download/nginx-$NGINX_VERSION.tar.gz
+  cd "$BUILD_DIR/passenger-$PASS_VERSION"
+  tar xf $BUILD_DIR/nginx.tar.gz
+
+  PREFIX=/opt/ood/ondemand/root
+  NGINX_DATADIR=$PREFIX/usr/share/nginx
+  NGINX_CONFDIR=$PREFIX/etc/nginx
+  NGINX_HOME=/var/lib/ondemand-nginx
+  NGINX_HOME_TMP=$NGINX_HOME/tmp
+  NGINX_LOGDIR=/var/log/ondemand-nginx
+  BASE_CCOPTS='-g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic'
```

Comment:
Let's replace these options `-m64 -mtune=generic` with the following `-march=native -mtune=native`:

```diff
-  BASE_CCOPTS='-g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic'
+  BASE_CCOPTS='-g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -march=native -mtune=native'
```